### DISCUSSION TITLE
MenuItem multiple line dropdown?

### DISCUSSION THREAD

**Discussion by rseydler** [iTwin/iTwinUI-react#179] [General]
Hi I'm a bit new to all of this so please forgive me for any dumb questions. 

I am using the MenuItem to return a list of Bentley connected projects. Due to the length of the project name + project description it will not fit in the MenuItem dropdown. I was hoping to split the dropdown items into two separate lines, one line for project name, then project description on the next line.
![image](https://user-images.githubusercontent.com/82630635/123880087-4b2c3c00-d985-11eb-82a2-07ce0deb5cf1.png)
What would be the best way to achieve that?
Thanks!

**Comment by bentleyvk** [comment]
Hi, good news for you! We are currently working on this https://github.com/iTwin/iTwinUI/pull/167. After that PR is done, we will add sublabels in iTwinUI-React.
![image](https://user-images.githubusercontent.com/36186912/123906731-6135fe80-d97d-11eb-9733-d1bf9d27c79b.png)

**Comment by mayank99** [comment]
Converting this into a feature request to track #205

**Reply by mayank99** [reply]
This is now available in iTwinUI-react 1.12.0 🎉

See example: https://itwin.github.io/iTwinUI-react/?path=/story/core-dropdownmenu--with-sublabels
